### ISSUE TITLE
Text colour option

### ISSUE THREAD

**Issue by gorillanajs** [evilprince2009/DotFetch#1]
Is it possible to add different colours to the text and ASCII image? Trying to figure out how but no dice.

**Comment by evilprince2009** [comment]
> Is it possible to add different colours to the text and ASCII image? Trying to figure out how but no dice.

Yes. You need to have some basic familiarity with shell color customization. 
Go through the links below
- https://www.shellhacks.com/bash-colors/
- https://stackoverflow.com/questions/5947742/how-to-change-the-output-color-of-echo-in-linux
These will help you.

**Comment by gorillanajs** [comment]
Thanks!!! Just what I was looking for. BrMagnusOn 9 Nov 2022, at 16:36, Ibne Nahian ***@***.***> wrote:﻿

Is it possible to add different colours to the text and ASCII image? Trying to figure out how but no dice.

Yes. You need to have some basic familiarity with shell color customization.
Go through the links below

https://www.shellhacks.com/bash-colors/
https://stackoverflow.com/questions/5947742/how-to-change-the-output-color-of-echo-in-linux
These will help you.
